doc/dev/general.md

```diff
@@ -49,20 +49,34 @@ but only generic tools, i.e. `kubectl` and `helm`.
 the chosen foundation, and assuming that the cluster does not come
 with Helm Tiller pre-installed.
 
-### Deployment and Tear-down
+### Deployment and Tear-down - cf-operator
 
 ```shell
 helm install --name cf-operator \
      --namespace cfo \
      --set "global.operator.watchNamespace=kubecf" \
-     https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.1%2B92.g77e53fda.tgz
+     https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.2-147.gb88e4296.tgz
 ```
 
 In the example above, version 0.4.1 of the operator was used. Look
 into the `cf_operator` section of the top-level `def.bzl` file to find
 the version of the operator validated against the current kubecf
 master.
 
+**Note:**
+> The above `helm install`, will generate two controllers into `cfo` namespace. The first
```

Comment:
There should be no comma before `will`.

This sentence can quickly become out-of-date. How about stating that there will be many controllers divided into multiple pods? cf-operator itself is not a single controller.

Reply:
fixed